test/e2e/apps/broadcastjob.go

```diff
@@ -82,7 +82,7 @@ var _ = SIGDescribe("BroadcastJob", func() {
 
 	framework.KruiseDescribe("BroadcastJob dispatching", func() {
 
-		ginkgo.It("succeeds for parallelism < number of node", func() {
+		framework.ConformanceIt("succeeds for parallelism < number of node", func() {
```

Comment:
why change the test to conformance test ?

Reply:
I think the e2e framework is based on k8s e2e framework. But we should not use ConformanceIt here.Need fix it